core/modules/system/css/messages.theme.css

```diff
@@ -23,7 +23,7 @@
   height: 100%;
   background-position: center .625em;
   background-repeat: no-repeat;
-  background-size: 1.5em;
+  background-size: 32px;
```

Comment:
I think we should keep to `em` sizing for consistency within Basis, which uses em for everything I believe.

Reply:
Nevermind, this is in `messages.theme.css`, not Basis. px is probably fine.

Reply:
neverminding.